lib/domains/character/CharacterFactory.tsx

```diff
@@ -37,6 +37,34 @@ export const CharacterFactory = {
     return {
       ...newCharacter,
       id: Id.generate(),
+            pages: newCharacter.pages.map((page) => {
```

Comment:
nit: I think there's a formatting issue here

Reply:
I'm not seeing it, but I'm happy to correct.